randovania/games/prime_hunters/pickup_database/pickup-database.json

```diff
@@ -13,31 +13,9 @@
             "hint_details": [
                 "a ",
                 "beam"
-            ]
-        },
-        "beam_related": {
-            "long_name": "Beam Related",
-            "hint_details": [
-                "a ",
-                "beam-related upgrade"
-            ],
-            "hidden": true
-        },
-        "missile_related": {
-            "long_name": "Missile Related",
-            "hint_details": [
-                "a ",
-                "missile-related upgrade"
             ],
             "hidden": true
         },
```

Comment:
the "a key" category soudns kinda... weird? especially since it has a lot of overlap with "alimbic artifact"